Edits to reflect new Sedai logo, marketing content and an out of the box dashboard (#1253)
DataDog/integrations-extras: sedai/README.md

## Overview

> Sedai is an autonomous cloud platform that proactively manages production environments to prevent issues and improve availability, performance, and cloud costs. As an intelligent autopilot for SREs, Sedai independently detects, prioritizes, and analyzes monitoring data for opportunities to safely and autonomously act in production - all without thresholds or human intervention. Learn more and try for free at [Sedai.io][1].

Reviewer: Curious about what the word opportunities is referring to here? Also, we generally try and stay away from including marketing/sales language in free integrations so I'd recommend removing the learn more and try for free sentence.

> Enable this integration to receive notifications in Datadog about actions Sedai autonomously executes in your production environments.

Reviewer: ```suggestion
Enable this integration to receive notifications in Datadog about actions that Sedai autonomously executes in your production environments.
```

### How it Works

* **Agentless:** Seamlessly connects to your cloud accounts and automatically discovers and understands production environments.

* **Configuration-free:** Easily connects to Datadog API and other APM/Observability platforms, and intelligently identifies, prioritizes and learns metric behavior.

> * **Proactive Actions:** Safely acts in production on your behalf to ensure resources avoid availability issues and run optimally at all times.

Reviewer: ```suggestion
* **Proactive Actions:** Safely acts in production on your behalf to ensure that resources avoid availability issues and run optimally at all times.
```

## Setup

In Sedai:

* Navigate to Settings > Integrations and add your Datadog account.
* Under Settings > Notifications, [select which notifications][3] you want to send to Datadog. 

## Data Collected

> This integration does not collect metrics or service checks, and does not support events.

Author: Sure thanks..

## Overview

Sedai is an autonomous cloud platform that proactively manages production environments to prevent issues and improve availability, performance, and cloud costs. As an intelligent autopilot for SREs, Sedai independently detects, prioritizes, and analyzes monitoring data to safely and autonomously act in production without thresholds.

Enable this integration to receive notifications in Datadog about actions that Sedai autonomously executes in your production environments.

### How it Works

* **Agentless:** Seamlessly connects to your cloud accounts and automatically discovers and understands production environments.

> * **Configuration-free:** Easily connects to Datadog API and other APM/Observability platforms, and intelligently identifies, prioritizes and learns metric behavior.

Reviewer: ```suggestion
* **Configuration-free:** Easily connects to Datadog API and intelligently identifies, prioritizes and learns metric behavior.
```

* **Proactive Actions:** Safely acts in production on your behalf to ensure that resources avoid availability issues and run optimally at all times.

## Setup

In Sedai:

> * Navigate to Settings > Notifications > Add Integration > DataDog Icon 

Reviewer: ```suggestion
* Navigate to Settings > Notifications > Add Integration > Datadog Icon 
```

> ![DataDog Notification Integration](https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/DataDog_Notification_Integration.png)

Reviewer 2: ```suggestion

![DataDog Notification Integration][3]
...
[3]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/DataDog_Notification_Integration.png)
```

Insert a newline so that the images aren't inline with the previous line of text. Also, can we move the image URLs to the reference links section?

Author: Sure, done and PR is updated.

> * Enter a Nick Name and the API Key for your DataDog account, Enable it and Test the integration.

Reviewer: ```suggestion
* Enter a nickname and the API Key for your Datadog account, enable it, and test the integration.
```

![DataDog Notification Integration](images/Add_DataDog_Channel.png)

> * Once the Test is verified to be working, click Save.

Reviewer: ```suggestion
* Once the test is verified to be working, click Save.
```

> ![DataDog Notification Integration](https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/Add_DataDog_Channel.png)

Reviewer 2: ```suggestion

![DataDog Notification Integration](https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/Add_DataDog_Channel.png)
```

> ![Working DataDog Integration](https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/Add_DataDog_Channel-Working_REC.png)

Reviewer 2: ```suggestion

![Working DataDog Integration](https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/Add_DataDog_Channel-Working_REC.png)
```

Author: PR Updated with changes for review comments

* Under Settings > Notifications, [select which notifications][2] you want to send to Datadog. 
![Enable DataDog Notifications](images/Enable_Notifications.png)

## Data Collected

> This integration does not collect metrics or service checks, and does not support events.

Reviewer: Apologies if we've already discussed this, but isn't Sedai sending data into Datadog through events? If not, what kind of data is being sent in?

Author: Hi @anawishnoff,
Yes, you are right - we send in Notifications on Sedai events that the user has enabled in set up steps to Datadog through the Events API.
Do you think the following line would do..? If not, could you please share your suggestion/recommendation..?

"This integration sends notifications on subscribed events from Sedai to Datadog."

> ![Enable DataDog Notifications](https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/Enable_Notifications.png)

Reviewer 2: ```suggestion

![Enable DataDog Notifications](https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/Enable_Notifications.png)
```

Author: PR Updated with changes for review comments

## Overview

Sedai is an autonomous cloud platform that proactively manages production environments to prevent issues and improve availability, performance, and cloud costs. As an intelligent autopilot for SREs, Sedai independently detects, prioritizes, and analyzes monitoring data to safely and autonomously act in production without thresholds.

Enable this integration to receive notifications in Datadog about actions that Sedai autonomously executes in your production environments.

### How it Works

* **Agentless:** Seamlessly connects to your cloud accounts and automatically discovers and understands production environments.

* **Configuration-free:** Easily connects to Datadog API and intelligently identifies, prioritizes and learns metric behavior.

* **Proactive Actions:** Safely acts in production on your behalf to ensure that resources avoid availability issues and run optimally at all times.

## Setup

In Sedai:

1. Navigate to Settings > Notifications > Add Integration > Datadog Icon

   ![Add Datadog Integration][3]

> 2. Enter a nickname and the API Key for your Datadog account, enable it and test the integration.

Reviewer 3: ```suggestion
2. Enter a nickname and the API key for your Datadog account. Enable and test the integration.
```

   ![Setup Datadog API Key][4]

3. Once the test is verified to be working, click Save.

   ![Save Working Datadog Integration][5]

4. Under Settings > Notifications, [select which notifications][2] you want to send to Datadog. 

   ![Enable Datadog Notifications][6]

## Data Collected

This integration sends events into Datadog.

## Support

> For help with this integration, please email [support@sedai.io][1].

Reviewer 3: ```suggestion
For help with this integration, contact [Datadog support][1].
```

Author: Hi @cswatt ,
Can you please confirm for support email, I should use Datadog's support email
 I've referred to other Readme(s) in the repo (Purefa, Postman  for example) and the support section shows the integration providers email, not that of Datadog.

> [1]: mailto:support@sedai.io

Reviewer 3: ```suggestion
[1]: https://docs.datadoghq.com/help/
```
